jump/src/process.rs

```diff
@@ -63,9 +63,20 @@ impl EnvVars {
                     removals.insert(name.to_owned());
                 }
                 EnvVar::RemoveMatching(regex) => {
-                    for (name, _) in env::vars() {
-                        if regex.is_match(name.as_str()) {
-                            removals.insert(name.into());
+                    for (name, _) in env::vars_os() {
```

Comment:
Confirmed the new unit tests fail on Windows and Linux prior to this fix and the new example IT in python/test.sh fails on Linux.